### PR TITLE
Release notes for 1.13.2

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Release notes for 1.13.2

*Test Results:*
No test needed. 

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on signing off your commits, please check [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/CONTRIBUTING.md#sign-your-work).

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-release-notes-1.13.2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Open Distro for Elasticsearch 1.13.1 Release Notes
```
Comment (gaiksaya) [inline release-notes/opendistro-for-elasticsearch-release-notes-1.13.2.md:1]: 1.13.2 :)

File: release-notes/opendistro-for-elasticsearch-release-notes-1.13.2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Release Engineering
+* Make sure replicas matches in values.yaml and README. Thanks @yardenshoham ([#686](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/686))
+* Indentation bug fix in values.yaml : Security Configuration. Thanks @mdiver ([#688](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/688))
+
+
+
+## MAINTENANCE
+
+### Index Management
+* Bump version to 1.13.1.0 ([#427](https://github.com/opendistro-for-elasticsearch/index-management/pull/427))
```
Comment (gaiksaya) [inline release-notes/opendistro-for-elasticsearch-release-notes-1.13.2.md:26]: Version to 1.13.2 PR says the same

Comment (sreekarjami) [reply release-notes/opendistro-for-elasticsearch-release-notes-1.13.2.md]: Changed.

**Comment by stockholmux** [comment]
LGTM @sreekarjami !

**Review by aetter — APPROVED** [review]

**Review by stockholmux — COMMENTED** [review]
LGTM

**Review by stockholmux — APPROVED** [review]

**Pull request merged**